ORG-ADMINS

```diff
@@ -1,7 +1,9 @@
-# Flux Oversight Committee
+# Flux Org Admins
 
 | Name | Slack handle | GitHub handle | Affiliation |
 | -- | -- | -- | -- |
+| Chris Aniszczyk | `caniszczyk` | `@caniszczyk` | CNCF |
 | Hidde Beydals | `hidde` | `@hiddeco` | Weaveworks  |
 | Stefan Prodan | `stefanprodan` | `@stefanprodan` | Weaveworks  |
 | Michael Bridgen | `Michael Bridgen` | `@squaremo` | Weaveworks  |
+| The Linux Foundation | - | `thelinuxfoundation` | The Linux Foundation |
```

Comment:
I remember @dholbach said @squaremo was OK with being removed in the earlier draft of this PR (when `fluxcd/flux2` repo maintainers were proposed as the project escalation point). 

However, now this PR now has separated Core Maintainers from `fluxcd/flux2` repo maintainers, and there is no longer a requirement that Core maintainers be listed in other `fluxcd` GitHub org repo MAINTAINERS files. Instead it's the other way around, granting implicit maintainership in other Flux repos where listed:

> Maintainership in the [CORE-MAINTAINERS file](https://github.com/fluxcd/community/blob/main/CORE-MAINTAINERS) trickles down to all other Flux 2-related repositories which means that maintainers mentioned there are also maintainers in all other repositories.
>
> In addition to maintaining `flux2` and Flux 2-related repositories, this team serves as escalation point for the overall project, and anything not easily managed by the Maintainers of each Git repository.

Therefore @squaremo could remain as one of the Flux Core Maintainers, which I would love to see.

Reply:
No, this is not quite right.

We have

1. The Core Maintainers, which are maintainers of Flux2. They are escalation point for a lot of questions - as layed out in this PR.
2. The Org Admins. These are folks that have elevated access to the Flux org in GH. Not everyone in the Flux2 maintainers group is comfortable with having them all. The `ORG-ADMINS` team you commented on just documents the status quo (adding CNCF and LF, so everyone who has access - this is fully transparent now).

See the list of follow-up items on the PR at the top: we will a) review the list after this PR is merged and see if we have new applicants and b) revisit the team annually.

Reply:
I wonder whether we should add some kind of a summary to the README or the PROCESS document that just lays out the different groups we have defined: org admins, core maintainers, individual repo maintainers, ..., each accompanied by a one-sentence description and link to the detailed description of the particular group's responsibilities etc.

Reply:
https://github.com/dholbach/community/blob/disolve-oversight-committee/community-roles.md#roles would be a great place.

Let's create the Org Admin team in GH once this is approved and add it to the list. It's right at the top so should be self explanatory - wdyt?